mods/ra/sequences/structures.yaml

```diff
@@ -93,7 +96,7 @@ proc:
 		Length: *
 	dead: procdead
 		Start: 0
-		ZOffset: -1c511
```

Comment:
Won't this cause the refinery's death anim to display on top of husks, for example?